README.md

```diff
@@ -97,9 +104,26 @@ To link your Vue components to their equivalent you created in Storyblok:
 
 ### 2. Getting Storyblok Stories and listen to Visual Editor events
 
-#### Composition API
+#### Composition API without `@nuxtjs/composition-api`
+
+> You don't need to install [@nuxtjs/composition-api](https://composition-api.nuxtjs.org/) if you're in the latest versions of Nuxt 2, as it comes with Vue 2.7 with Composition API support out of the box.
+
+The simplest way is by using the `useStoryblok` one-liner composable, which uses the [useFetch from @nuxtjs/composition-api](https://composition-api.nuxtjs.org/lifecycle/useFetch) under the hood:
```

Comment:
I realize I would skip the useFetch explanation, as it might be confusing for the user